lib/screens/tree_list/tree_list_view_model.dart

```diff
@@ -76,4 +84,9 @@ abstract class TreeListViewModelBase with Store, ViewModel {
     fetch(startRow: treeStore.countTreeList() + 1, nbRows: 20)
         .then((_) => isLoadingTrees = false);
   }
-}
+
+  void connectionChanged(dynamic hasConnection) {
```

Comment:
Use specific type instead of dynamic

Reply:
The argument type 'void Function(bool)' can't be assigned to the parameter type 'void Function(dynamic)?'.
StreamSubscription<dynamic> listen( void Function(dynamic)? onData, { Function? onError, void Function()? onDone, bool? cancelOnError, })